src/script.c

```diff
@@ -159,6 +159,11 @@ btc_bool btc_script_get_ops(const cstring* script_in, vector* ops_out)
             continue;
         }
 
+        // don't alloc a push buffer if there is no more data available
+        if (buf.len == 0 || data_len > buf.len) {
```

Comment:
Looks like data_len is always `> 0` right? `if (data_len > buf.len)` should be enough then.

Reply:
Could `deser_bytes` not de-serialize a 0 byte?